bleak/backends/service.py

```diff
@@ -79,46 +85,64 @@ def __init__(self):
 
     def __getitem__(
         self, item: Union[str, int, UUID]
-    ) -> Union[BleakGATTService, BleakGATTCharacteristic, BleakGATTDescriptor]:
+    ) -> Union[BleakGATTService, BleakGATTCharacteristic, BleakGATTDescriptor, None]:
```

Comment:
FWIW, I think I have been using this patter in Bleak.

    Optional[Union[BleakGATTService, BleakGATTCharacteristic, BleakGATTDescriptor]]

Reply:
Did not know about `Optional` and it being the preferred solution. Thank you!